go.mod

```diff
@@ -3,9 +3,9 @@ module github.com/cert-manager/approver-policy
 go 1.24.0
```

Comment:
Looks like you were right @erikgb

Reply:
Yeah, I have been fighting a lot with Depedabot (and Renovate) and Go toolchains. I still don't see the use case for Go toolchains, and think they are confusing.....